pyracing/client.py

```diff
@@ -42,6 +42,31 @@ def __init__(self, username: str, password: str):
         self.password = password
         self.session = httpx.AsyncClient()
 
+    def _rename_numerical_keys(self, response_item, mapping):
+        """ Returns a version of 'response_item' where all numerical keys have 
+        been renamed to readable strings by applying 'mapping'.
+        """
+        if not isinstance(response_item, dict):
+            logger.warning('The response_item you provided does not appear to be a '
+                'dictionary. Check if you are calling the '
+                '_rename_numerical_keys method correctly.')
+        
+        if not isinstance(mapping, dict):
+            logger.warning('The mapping you provided does not appear to be a '
```

Comment:
Same as above, this log message isn't super helpful here. Also for this one and the one above, we should return after logging as it will not work if it fails any of these. 

Honestly, we could maybe even make them the same if statement with 
`not isinstance(response_item, dict) or not isinstance(mapping, dict):` since there isn't much difference in why they would fail.

Reply:
See above